framework/validators/FileValidator.php

```diff
@@ -80,6 +80,16 @@ class FileValidator extends Validator
      * @see tooMany for the customized message when too many files are uploaded.
      */
     public $maxFiles = 1;
+
+    /**
+     * @var int the minimum file count the given attribute can hold.
+     * Defaults to 1, meaning single file upload. By defining a higher number,
+     * multiple uploads become possible. Setting it to `0` means there is no limit on
+     * the number of files that can be uploaded simultaneously.
+     *
+     * @see tooFew for the customized message when too few files are uploaded.
+     */
+    public $minFiles = 1;
```

Comment:
I don't think it should be 1 by default since it will break BC by disallowing not uploading a file.

Reply:
Fix it